glslc/src/file_compiler.cc

```diff
@@ -104,8 +133,9 @@ bool FileCompiler::CompileShaderFile(const std::string& input_file,
   // later, if the handler is instantiated to dump as normal compilation output,
   // and the original compilation output should be blocked. Otherwise it won't
   // be touched. The main output stream dumps this string_piece later.
-  string_piece compilation_output(result.GetData(),
-                                  result.GetData() + result.GetLength());
+  string_piece compilation_output(
```

Comment:
Yes we do because OutputElementType can be a uint32_t and you need the reinterpret cast to change the pointer type.